test/e2e/scan/scan_utils.go

```diff
@@ -84,3 +94,123 @@ func RunScanCommand(terrascanBinaryPath, relGoldenFilePath string, exitCode int,
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	return session, goldenFileAbsPath
 }
+
+// GetSarifGoldenString gives out golden sarif format string
+func GetSarifGoldenString(template, terrascanVersion string, absfilepath string) string {
+	return fmt.Sprintf(template, terrascanVersion, fmt.Sprintf("file://%s", absfilepath))
+}
+
+// SarifTemplateAWSAMIViolation string
+const SarifTemplateAWSAMIViolation = `{
```

Comment:
I was wondering if we can use golden text files instead of a template. Is it possible?

Reply:
I think It'll unnecessary add another step of file processing. Maybe we can take this point up in some other refactor phase as such templates are being used in other places as well.